doc/esmvalcore/utils.rst

```diff
@@ -0,0 +1,23 @@
+.. _utils:
```

Comment:
Can't we just replace this with a link to the yaml documentation on the web?

Reply:
I like the idea of a local mini guide, this was initially plugged in by @ledm and I think it's useful for lazy scientists that don't want to sift through the yaml documentation